pyproject.toml

```diff
@@ -113,9 +113,9 @@ lines-between-types = 1 # Separate import/from with 1 line
 convention = "pep257"
 
 [tool.setuptools.dynamic]
-dependencies = {file = "requirements.txt"}
-optional-dependencies.docs = {file = [".config/docs-requirements.txt"]}
-optional-dependencies.test = {file = [".config/requirements-test.txt"]}
+dependencies = {file = ".config/requirements.in"}
+optional-dependencies.docs = {file = [".config/docs-requirements.in"]}
```

Comment:
@NilashishC just to keep the same pattern as other projects, could you please rename to `requirements-docs.in` and `requirements-test.in`